index.js

```diff
@@ -20,8 +20,52 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    Recipe.create({title: "Asian Glazed Chicken Thighs",
+    level: "Amateur Chef",
+    ingredients: [
+      "1/2 cup rice vinegar",
+      "5 tablespoons honey",
+      "1/3 cup soy sauce (such as Silver Swan®)",
+      "1/4 cup Asian (toasted) sesame oil",
+      "3 tablespoons Asian chili garlic sauce",
+      "3 tablespoons minced garlic",
+      "salt to taste",
+      "8 skinless, boneless chicken thighs"
+    ],
+    cuisine: "Asian",
+    dishType: "main_course",
+    image: "https://images.media-allrecipes.com/userphotos/720x405/815964.jpg",
+    duration: 40,
+    creator: "Chef LePapu"})
+      .then((result) => {
+        console.log(result.title)
+      })
+      .catch ((err) => {
+        console.log('Loser!', err)
+      })
+
+    Recipe.insertMany(data)
+    .then((thing) => {
```

Comment:
using thing is fine when you have no idea what you are getting xD but after you know it is the same recipes you created you can change this parameters to be descriptive of the content. Something like recipes or insertedRecipes. You may be using this variables later in some additional coding and for readability it is much better.